modules/update-preparing-evaluate-apirequestcount.adoc

```diff
@@ -37,7 +37,7 @@ installplans.v1alpha1.operators.coreos.com                     93
 ====
 You can safely ignore the following entries that appear in the results:
 
-* The `system:serviceaccount:kube-system:generic-garbage-collector` user might appear in the results because it walks through all registered APIs searching for resources to remove.
+* The `system:serviceaccount:kube-system:generic-garbage-collector` and `system:serviceaccount:kube-system:namespace-controller` users might appear in the results because they walk through all registered APIs searching for resources to remove.
```

Comment:
```suggestion
* The `system:serviceaccount:kube-system:generic-garbage-collector` and the `system:serviceaccount:kube-system:namespace-controller` users might appear in the results because they walk through all registered APIs searching for resources to remove.
```